internal/output/githubannotation.go

```diff
@@ -0,0 +1,46 @@
+package output
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/osv-scanner/pkg/models"
+)
+
+// PrintGHAnnotationReport prints Github specific annotations to outputWriter
+func PrintGHAnnotationReport(vulnResult *models.VulnerabilityResults, outputWriter io.Writer) error {
+	flattened := vulnResult.Flatten()
+
+	// TODO: Also support last affected
+	groupFixedVersions := GroupFixedVersions(flattened)
+	workingDir, workingDirErr := os.Getwd()
```

Comment:
How common is it for us to get a workingDirErr? Would it make sense to just panic instead to make the code simpler if this is extremely rare?

Reply:
Changed it to panic now